sig-service-catalog/charter.md

```diff
@@ -0,0 +1,159 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open
+Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI). It allows
+application developers the ability to provision and consume cloud services
+natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the [OSBAPI
+  specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md).
+- Provide a UX for Kubernetes users that is consistent with both the OSB API
+  specification and traditional Kubernetes user interactions.
+- Align with the OSBAPI specification as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
```

Comment:
Would you mind providing a link to your SIG's sigs.yaml file as well as a list of any code/binaries/services this SIG owns? We recently added an[ example SIG charter template](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-charter-template.md) if that's helpful to look at.

Reply:
Will do!

We do have a list further down of all the code/services/etc that we own. Do you need me to move it up into the Scope section?

https://github.com/carolynvs/community/blob/sig-service-catalog-charter/sig-service-catalog/charter.md#assets

Reply:
yeah if you could make it look like that template, that would be awesome. thanks!!

Reply:
done!